default_isce22.bash

```diff
@@ -0,0 +1,92 @@
+# vim: set filetype=sh:
```

Comment:
Again, anything that is specific to our install of the code (ie. the directory structure, aliases, etc.) shouldn't be under version control, as users may not want or use an indexical setup to us.

I would recommend pruning this .bash file down to only those lines that are critical for the code to run properly (ie. remove anything that has to do with the way we structure our file system).